app/sidekiq/evss/disability_compensation_form/form526_document_upload_failure_email.rb

```diff
@@ -62,17 +55,28 @@ class Form526DocumentUploadFailureEmail < Job
           }
         )
         raise e
+      ensure
+        StatsD.increment("#{STATSD_METRIC_PREFIX}.exhausted")
+
+        cl = caller_locations.first
```

Comment:
Whatttt I learned something today didn't even know what this was

Reply:
same